app/posts/ecf-v2/initial-release-of-ecf-2.md

```diff
@@ -0,0 +1,82 @@
+
+## How we decided what we would include in the MVP
+
+We decided what we needed to include for the MVP for the new digital services by considering:
+* what the current ECF services do
+* the core purposes of the ECF digital services
+* our ideas for improvements to these services based on the problems we discovered
+* how we could increase flexibility, self-sustainability and autonomy for these services
+
+## The MVP for the new digital services for ECF 
+
+We've aligned everything we want to do or build for MVP to the core purposes for [Teacher Continuing Professional Development programme](https://teacher-cpd.design-history.education.gov.uk/cross-programme/our-refreshed-vision/).
+
+Here is a map showing what we hope to build for the MVP of the new digital services.
+
+![A map showing what we want to build for the MVP of the new digital services. It includes building a new Lead Provider API, a database just for ECF with an improved model, a service for appropriate bodies to check training data, joint support and finance tooling for internal users and a service for schools to register early career teachers and mentors.](app/images/ecf-2/initial-release-of-ecf-2/mvp-map.png)
+
+### Capture and manage the data to enable teachers and leaders to take up ECF programmes 
+
+To meet this purpose in our initial release, we will: 
+* Build a more intuitive journey in the digital service for schools to register ECTs and mentors with DfE, that also helps them understand their ECF responsibilities  
+* Build an improved and simplified data model for a separated database from NPQ that better represents reality (e.g. participants that span multiple contracts)  
+* Build an easier way for schools to access the service (e.g. DFE sign in)  
+* Migrate and clean the data we need to sustain the registration and training of ECTs and mentors  
+* Create improved API documentation for lead providers in the style of NPQ  
+* Build a service for appropriate bodies to check their training data 
+* Create improved ways for schools and lead providers to make changes to ECT and mentor records, rather than support or developers  
+* Include in the data model reliable tracking of more changes to data, such as eligibility for training or funding 
+* Create an API for lead providers with the same endpoints, so we don't cause too much work for them  
+* Build improved support tooling and processes, in the same place as finance tooling
+* Make changes to the services to follow changes to ECF policy in 2025  
+* Enable grant funding to schools to backfill mentors while they are completing training
+
+### Capture and manage ECF data to facilitate payments to providers and enable effective contract management 
+
+To meet this purpose in our initial release, we will:
+* Build a way for payments to be calculated for lead provider training, so lead providers are paid the right amount  
```

Comment:
"Rebuild a way for payments to be calculated for lead provider training using the improved data model"